content/blog/2016/2016-08-22-ewm-stable-release.adoc

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "GSoC: External Workspace Manager for Pipeline is released"
+tags:
+- pipeline
+- plugins
+- external-workspace-manager
+- gsoc
+author: alexsomai
+---
+
+This blog post is the last one from the series of
+link:https://summerofcode.withgoogle.com/[Google Summer of Code] 2016, External Workspace Manager Plugin project.
+The previous posts are:
+
+* link:/blog/2016/05/23/external-workspace-manager-plugin/[Introductory blog post]
+* link:/blog/2016/06/30/ewm-alpha-version/[Alpha release announcement]
+* link:/blog/2016/08/09/ewm-beta-version/[Beta release announcement]
+
+By this I'd like to announce that the External Workspace Manager Plugin version 1.0.0 has been released!
```

Comment:
> By this I'd like to announce

The phrase "By this" I *believe* is referring to this blog post, but it is a confusing sentence. I suggest:

> In this post I would like to announce the 1.0.0 release of the External Workspace Manager Plugin version to the main update center.